### PR TITLE
[Relay][OP] Add fast_erf implementation

### PR DESCRIPTION
**Pull request by icemelon9**
Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

@anijain2305

### CODE REVIEW

File: topi/include/topi/elemwise.h [hunk 1]
```diff
@@ -461,5 +462,75 @@ inline Tensor fast_exp(const Tensor& x,
...
+
+  // The monomial coefficients of the denominator polynomial (even).
+  auto beta_0 = make_const(DataType::Float(32), -1.42647390514189e-02f);
+  auto beta_2 = make_const(DataType::Float(32), -7.37332916720468e-03f);
+  auto beta_4 = make_const(DataType::Float(32), -1.68282697438203e-03f);
+  auto beta_6 = make_const(DataType::Float(32), -2.13374055278905e-04f);
+  auto beta_8 = make_const(DataType::Float(32), -1.45660718464996e-05f);
+
+  return compute(data->shape, [&](const Array<Var> &i) {
+      // clamp x
```
Comment (anijain2305) [inline topi/include/topi/elemwise.h:492]: Should this be 2 spaces instead of 4?

Comment (icemelon9) [reply topi/include/topi/elemwise.h]: fixed

**Comment by anijain2305** [comment]
Do you know why windows build is failing?

**Comment by icemelon9** [comment]
Not really. The log shows the error
```
2020-04-05T22:36:45.4562113Z D:\a\incubator-tvm\incubator-tvm\topi\src\topi.cc : fatal error C1128: number of sections exceeded object file format limit: compile with /bigobj [D:\a\incubator-tvm\incubator-tvm\build.common\tvm_topi.vcxproj]
```

But in TVM cmake, we already set the `/bigobj` option in the [cmake](https://github.com/apache/incubator-tvm/blob/master/CMakeLists.txt#L90).

@tqchen Do you have a better idea?

**Comment by tqchen** [comment]
Let us try to break topi.cc into multiple files grouped by their .h

**Comment by icemelon9** [comment]
@anijain2305 ci is now green. Could you take a look again?

**Comment by anijain2305** [comment]
Thanks @icemelon9 @tqchen This is merged

**Pull request merged**